packages/entities/entities-consumer-groups/src/components/ConsumerGroupList.cy.ts

```diff
@@ -310,10 +310,10 @@ describe('<ConsumerGroupList />', () => {
         props: {
           cacheIdentifier,
           config: baseConfigKM,
-          canCreate: () => {},
-          canEdit: () => {},
-          canDelete: () => {},
-          canRetrieve: () => {},
+          canCreate: () => { },
```

Comment:
are those changes coming from lint rules ?

Reply:
Eh no, my VSC formatted them this way for some reason but I backed out these formatting changes now

Reply:
I take that back, those are the lint rules